app/models/mixins/reportable_mixin.rb

```diff
@@ -93,23 +93,17 @@ def search(count = :all, options = {})
 
     # private
 
-    def get_include_for_find(report_option)
-      includes = report_option
-      if includes.kind_of?(Hash)
-        result = {}
-        includes.each do |k, v|
+    def get_include_for_find(includes)
+      case includes
+      when Hash
+        includes.each_with_object({}) do |(k, v), result|
           v[:include] = v["include"] if v["include"]
```

Comment:
By the way, this assignment looks like a bit of code smell to me, but I didn't want to change the behaviour of the method so I left it as is. Query methods (as in [CQRS](https://ru.wikipedia.org/wiki/CQRS)) generally should not mutate their arguments.